lib/capnp_c.h

```diff
@@ -22,6 +28,19 @@
 typedef intmax_t ssize_t;
 #endif
 
+// Macro UNUSED(x) silences compiler warnings about unused parameter or variable `x`.
+#ifndef UNUSED
+#define UNUSED(x) (void)(x)
+#endif
+
+// Cross-platform macro ALIGNED_(x) aligns a struct by `x` bits.
```

Comment:
OK. I misunderstood the reference I was using, sorry. I will update the comment to say bytes.

I'm compiling for a 32-bit ARM microcontroller, so pointers are 32 bits long and not enough on their own. Without the alignment attribute on [`capn_segment`](https://github.com/fluffysquirrels/c-capnproto/blob/1b25eae747ccf73472ed3984d283ab5a62389dff/lib/capnp_c.h#L126-L135), [`struct check_segment_alignment`](https://github.com/fluffysquirrels/c-capnproto/blob/1b25eae747ccf73472ed3984d283ab5a62389dff/lib/capn-malloc.c#L18-L25) in `lib/capn-malloc.c` was failing to compile. The comments there suggest that the alignment attribute is also required for x86, which uses 32-bit pointers, of course.

Are you happy for the alignment attribute to stay in?